### DISCUSSION TITLE
Motion detection on Hero 13

### DISCUSSION THREAD

**Discussion by aneedham26** [gopro/labs#1106] [General]
I can't seem to get Motion Detection to work on the Hero 13 more than once. Using the same settings on the Hero 12 it works fine, but on the 13 it records once and then will not record again. I'm using the Sept 24th version of labs. What I've noticed is that the text "0 difference shutter to cancel " disappears after the first recording. Using 4K 30 fps wide on both the 12 and the 13

**Comment by dnewman-gpsw** [comment]
It should now all work from https://gopro.github.io/labs/control/motion/  I changed the repeat from !R to !1R, so give the camera more time to finishing the capture.  It is something that I will investigate, to find out why there is this need.

**Reply by aneedham26** [reply]
OK, it seems I got it to work by using the QR control app for Android instead of the online site. Not sure why.
